examples/poll_inv_changes.py

```diff
@@ -0,0 +1,78 @@
+'''
+Script: Poll for changes to investigations and incidents 
+Version: 1.0
+
+Usage:
+    python poll_inv_changes.py 
+
+'''
+
+import pprint
+import time
+import sys
+import getpass
+
+from datetime import datetime
+from datetime import timedelta
+from pyexclient import WorkbenchClient
+from pyexclient.workbench import gt
+from pyexclient.workbench import notnull 
+from pyexclient.workbench import relationship 
+
+
+def authenticate():
+    '''
+    Prompt user for authentication info
+    '''
+    username = input("Enter Username: ")
+    password = getpass.getpass("Enter Password: ")
+    code = input("2FA Code: ")
+    xc = WorkbenchClient('https://workbench.expel.io', username=username, password=password, mfa_code=code)
+    return xc
+
+def get_inv_changes(xc, since):
+    '''
+    Method polls a few different endpoints for updates to their histories which indicate updates/changes in Workbench.
+    '''
+
+    for change in xc.investigative_action_histories.search(relationship('investigation.id', notnull()), created_at=gt(since.isoformat()) ):
+        if change.investigation is None:
```

Comment:
this check shouldn't be necessary given the not null filter you have in your search criteria.

Reply:
or will the `investigation` property pro-actively go and fetch the investigation?

Reply:
After reading the rest of the script, I think it's implied that the the getter on these properties are actually fetching the records from the API, but I don't remember seeing that mentioned in the pyexclient docs.  Did I miss that?

Reply:
Yes these properties go out and retrieve the record from the API. Sounds like an opportunity for us to improve docs!